packages/protocol/contracts/common/interfaces/IAccounts.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity ^0.5.3;
+
+
+interface IAccounts {
+  function isAccount(address) external view returns (bool);
+  function getAccountFromVoteSigner(address) external view returns (address);
+  function getAccountFromValidationSigner(address) external view returns (address);
+  function getValidationSignerFromAccount(address) external view returns (address);
```

Comment:
being that we are in the "Accounts" contract, i feel unnecessary to have the `FromAccount` suffix here; how about `getValidationSigner(account)`?